### ISSUE TITLE
TypeError after deleting layer: Cannot read property 'info' of undefined

### ISSUE THREAD

**Issue by karandesai-96** [Cloud-CV/Fabrik#108]
Whenever a layer is added in workspace and later deleted, I find this exception being thrown through the Chrome Console.

First traceback is while rendering the tooltip of `hoveredLayer` which would be undefined since it was deleted (I guess?):
```text
Uncaught TypeError: Cannot read property 'info' of undefined
    at Tooltip.render (bundle.js:44344)
    at bundle.js:16503
    at measureLifeCyclePerf (bundle.js:15783)
    at ReactCompositeComponentWrapper._renderValidatedComponentWithoutOwnerOrContext (bundle.js:16502)
    at ReactCompositeComponentWrapper._renderValidatedComponent (bundle.js:16529)
    at ReactCompositeComponentWrapper._updateRenderedComponent (bundle.js:16453)
    at ReactCompositeComponentWrapper._performComponentUpdate (bundle.js:16431)
    at ReactCompositeComponentWrapper.updateComponent (bundle.js:16352)
    at ReactCompositeComponentWrapper.receiveComponent (bundle.js:16254)
    at Object.receiveComponent (bundle.js:8423)
```

Next, when I hover on another layer, it throws a new exception:
```text
bundle.js:28035 Uncaught TypeError: Cannot read property 'info' of undefined
    at Content.changeHoveredLayer (bundle.js:28035)
    at Canvas.hoverLayerEvent (bundle.js:28862)
    at onMouseEnter (bundle.js:33113)
    at Object.ReactErrorUtils.invokeGuardedCallback (bundle.js:6822)
    at executeDispatch (bundle.js:6607)
    at Object.executeDispatchesInOrder (bundle.js:6630)
    at executeDispatchesAndRelease (bundle.js:6028)
    at executeDispatchesAndReleaseTopLevel (bundle.js:6039)
    at Array.forEach (<anonymous>)
    at forEachAccumulated (bundle.js:6921)
```

**Comment by karandesai-96** [comment]
I'm occupied with a few open PRs these days, but I'd want to see if I can solve it in a considerably brief amount of time. If not I'll write here again.

**Comment by virajprabhu** [comment]
@karandesai-96 Great, let us know if you can submit a fix.

**Comment by utsavgarg** [comment]
Fixed in #129
